server/gql/models/authorsBooks/index.js

```diff
@@ -35,12 +33,7 @@ const AuthorsBook = new GraphQLObjectType({
   interfaces: [nodeInterface],
   fields: () => ({
     ...getQueryFields(authorsBookFields, TYPE_ATTRIBUTES.isNonNull),
-    ...timestamps,
-    authors: {
```

Comment:
we don't need the authors relation here?

Reply:
nope